sdk/keyvault/microsoft-azure-keyvault-complete/pom.xml

```diff
@@ -17,7 +17,11 @@ the MIT License. See License.txt in the project root for license information. --
   <packaging>pom</packaging>
 
   <name>Microsoft Azure Key Vault SDK Complete</name>
-  <description>This encapsulate all modules of the Microsoft Azure Key Vault SDK.</description>
+  <description>This library has been replaced by new Azure SDKs, you can read about them at https://aka.ms/azsdkvalueprop. The latest libraries to interact with the Azure KeyVault service are:
+    1. https://search.maven.org/artifact/com.azure/azure-security-keyvault-keys
```

Comment:
Does the description maintain newlines or do they need to be added with special characters?

Reply:
That is actually something @g2vinay and were not sure about. Your comment made me do some research and found that we can use ${line.separator} as showed in [this StackOverflow comment](https://stackoverflow.com/a/16294529/4967034).